### DISCUSSION TITLE
Mirrors not exporting serial numbers

### DISCUSSION THREAD

**Discussion by rubenskuhl** [irrdnet/irrd#584] [Q&A] [answered]
For some sources that publish mirror of our data in their export folder, we export theirs as well. But we are not exporting serials, only dbs. Does anyone know why is that ? We do have class filter on, scope_filter on and journal off, and the aggregate effect might be causing this.

**Comment by mxsasha** [comment, marked as answer]
Yes, the issue here is the journal being switched off. Basically, your local IRRD instance does not know what the serial number is for its local data, as that would usually be counted in the journal. We don't export the latest serial seen from the mirror, because when you make local changes to the data from e.g. the scope filter, the serial would not increase even though the data has changed.
